soaks/bin/run_experiment.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-set -o errexit
+#set -o errexit # grep will exit 1 without a match, even with --count
 set -o pipefail
 set -o nounset
-#set -o xtrace
+set -o xtrace
```

Comment:
Did you mean to leave this uncommented?

Reply:
I did, although I'll flip it off next week if we don't have any weirdness from this area of the soak scripts.

Reply:
👍